testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/headers/ResponseCodeTestCase.java

```diff
@@ -103,15 +103,15 @@ public void test500() throws Exception {
         doContentTypeChecks(response, 500);
     }
 
-     @Test
+   /*  @Test
```

Comment:
Seems to be commented out in error.

Reply:
not really. this test is wrong and doesn't belong into basic testsuite.

Reply:
Note to self/others: The issue here is with this PR, testsuite/integration/basic no longer has bin/product.conf so this test can't pass, as that drives. The idea is to move it to smoke test suite as there bin/product.conf exists. This is basically a kind of image verification test, so Tomaz's idea is to move such things to smoke.

Reply:
Sounds good. Hopefully it won't get forgotten as commented out :-) What I was aiming at, is maybe an @Ignore would be better in a sense it will get noticed and acted upon.